test-data/unit/check-union-or-syntax.test

```diff
@@ -67,7 +67,6 @@ reveal_type(x)  # N: Revealed type is "builtins.list[Union[builtins.int, builtin
 [builtins fixtures/list.pyi]
 
 [case testUnionOrSyntaxWithQuotedFunctionTypes]
-# flags: --python-version 3.4
 from typing import Union
```

Comment:
We need to make sure --python-version is set to less than 3.10 here, since the whole point of the test is to make sure you can use PEP-604 syntax on older Python versions if it's quoted

```suggestion
# flags: --python-version 3.9
from typing import Union
```